components/create_comment/create_comment.jsx

```diff
@@ -354,7 +354,9 @@ export default class CreateComment extends React.PureComponent {
     }
 
     handleFileUploadChange = () => {
-        this.focusTextbox();
```

Comment:
Looks like we have the exact same code in `create_post/create_post.jsx` probably it has the same problem. (Not for now, but we can consider to have a base class for create_post.jsx and create_comment.jsx, because they share a lot of behavior and code).

Reply:
That's something I've wanted for a long time, but I haven't found a chance to go do that

Reply:
Maybe a help wanted ticket?

Reply:
@jespino You're correct about CreatePost. It should have the same behavior.